### PR TITLE
Skip first time full dill transfer

### PR DESCRIPTION
**Pull request by aam**
This should improve hotReloadInitialDevFSSyncMilliseconds and hotReloadInitialDevFSSyncAfterRelaunchMilliseconds benchmarks.

### CODE REVIEW

File: packages/flutter_tools/lib/src/devfs.dart [hunk 1]
```diff
@@ -509,15 +509,20 @@ class DevFS {
...
-          dirtyEntries[entryUri] = content;
-          numBytes += content.size;
+      // Don't send full kernel file that would overwrite what VM already
+      // started loading from.
+      if (!bundleFirstUpload) {
+        final String compiledBinary = compilerOutput?.outputFilename;
+        if (compiledBinary != null && compiledBinary.isNotEmpty) {
+          final Uri entryUri = fs.path.toUri(projectRootPath != null ?
+          fs.path.relative(pathToReload, from: projectRootPath) :
+          pathToReload);
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/devfs.dart]: nit: indentation here is confusing

**Comment by Hixie** [comment]
I assume the dill is now bundled in the debug-mode APK?

code LGTM.

File: packages/flutter_tools/lib/src/devfs.dart (hunk 1 above)
Comment (aam) [reply packages/flutter_tools/lib/src/devfs.dart]: Yeah, IJ auto-indenter took over as soon as added curly braces.

**Comment by aam** [comment]
> I assume the dill is now bundled in the debug-mode APK?

Right.

**Review by a-siva — APPROVED** [review]

**Pull request merged**